binary/integer.go

```diff
@@ -100,6 +101,10 @@ func S256(x *big.Int) *big.Int {
 	}
 }
 
+func Big256() *big.Int {
```

Comment:
we should just export the one above - ideally we'd like to only allocate once which is achieved by a var initialisation